### PR TITLE
Add module to decode admin proposals

### PR DESCRIPTION
**Pull request by mrice32**
This adds a module to the CLI tool that will print out information about admin proposals.

### CODE REVIEW

File: core/scripts/cli/admin.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    console.group();
+    console.log("Transaction", i);
+
+    const transaction = proposal.transactions[i];
+
+    // Give to and value.
+    console.log("To: ", transaction.to);
+    console.log("Value (in Wei): ", transaction.value);
+
+    if (!transaction.data || transaction.data.length === 0 || transaction.data === "0x") {
```
Comment (nicholaspai) [inline core/scripts/cli/admin.js]: in which cases would `transaction.data === "0x"`?

File: core/scripts/cli/admin.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      // No data -> simple ETH send.
+      console.log("Transaction is a simple ETH send (no data).");
+    } else {
+      // Txn data isn't empty -- attempt to decode.
+      const decodedTxn = abiDecoder.decodeMethod(transaction.data);
+      if (!decodedTxn) {
+        // Cannot decode txn, just give the user the raw data.
+        console.log("Cannot decode transaction (does not match any UMA Protocol Signauture.");
```
Comment (nicholaspai) [inline core/scripts/cli/admin.js]: nit: typo "Signauture"

File: core/scripts/cli/admin.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  const voting = await Voting.deployed();
+
+  // Search through pending requests to find active governor proposals.
+  const pendingRequests = await voting.getPendingRequests();
+  const adminRequests = [];
+  for (const pendingRequest of pendingRequests) {
+    const identifier = web3.utils.hexToUtf8(pendingRequest.identifier);
+    if (identifier.startsWith("Admin ")) {
+      // This is an admin proposal.
+      const id = parseInt(identifier.slice(6));
```
Comment (nicholaspai) [inline core/scripts/cli/admin.js]: not sure what existing style instructions are, but maybe make the hardcoded `6` programmatic as `const adminIdentifier = "Admin "; ...parseInt(identifier.slice(adminIdentifier.length))`

File: core/scripts/cli/admin.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  }
+  console.groupEnd();
+}
+
+async function viewAdminMenu(maxId) {
+  const prompts = [
+    {
+      type: "input",
+      name: "viewAdminMenu",
+      message: `Please enter an admin propsal id (0-${maxId}), Enter to see all active admin proposals, or 'exit' to quit.`
```
Comment (nicholaspai) [inline core/scripts/cli/admin.js]: nit typo `propsal`

File: core/scripts/cli/admin.js (hunk 3 above)
Comment (mrice32) [reply core/scripts/cli/admin.js]: SGTM -- good point.

File: core/scripts/cli/admin.js (hunk 1 above)
Comment (mrice32) [reply core/scripts/cli/admin.js]: When I was testing this, an empty data field (i.e. just an eth transfer) has a return value of "0x".

File: core/scripts/cli/admin.js (hunk 4 above)
Comment (mrice32) [reply core/scripts/cli/admin.js]: Done.

File: core/scripts/cli/admin.js (hunk 3 above)
Comment (mrice32) [reply core/scripts/cli/admin.js]: Done.

File: core/scripts/cli/admin.js (hunk 2 above)
Comment (mrice32) [reply core/scripts/cli/admin.js]: Done.

**Pull request merged**